Common/Data/Consolidators/ClassicRangeConsolidator.cs

```diff
@@ -20,22 +20,29 @@
 namespace QuantConnect.Data.Consolidators
 {
     /// <summary>
-    /// This consolidator can transform a stream of <see cref="IBaseData"/> instances into a stream of <see cref="RangeBar"/>
+    /// This consolidator can transform a stream of <see cref="IBaseData"/> instances into a stream of <see cref="RangeBar"/>.
+    /// The difference between this consolidator and <see cref="RangeConsolidator"/>, is that this last one creates intermediate/
+    /// phantom RangeBar's (RangeBar's with zero volume) if the price rises up or falls down by above/below two times the range 
```

Comment:
I think phantom bars are created if price rises/falls above/below the RangeSize, right? Not 2*RangeSize

Reply:
It looks like, but if the price rises above the `currentBar.Low + BarSize` but not above `currentBar.Low + 2 * BarSize`, it will only close one bar, and create another, but this last one won't be closed yet. In that case, both the bar that closed and the bar that was created are real bars. If the price would have risen above `currentBar.Low + 2 * BarSize` It woudln't have just closed one bar, but even created and closed another one, that last one bar is the one that would be phantom bar.